script.js

```diff
@@ -1,12 +1,434 @@
+const documentaryContainer = document.getElementById("documentary-series");
+const documentaryCard = createCardForEachShow(findDocumentary);
+documentaryContainer.appendChild(documentaryCard);
+
+//<-----------------------------------------dramas---------------------------->
+let findDramas = findGenres("Drama");
+const dramaContainer = document.getElementById("drama");
+const dramaCard = createCardForEachShow(findDramas);
+dramaContainer.appendChild(dramaCard);
+
+//<------------------------------------------action----------------------------->
+let findAction = findGenres("Action");
+const actionContainer = document.getElementById("action");
+const actionCard = createCardForEachShow(findAction);
+actionContainer.appendChild(actionCard);
+
+//<----------------------------------------comedy--------------------------->
+let findComedy = findGenres("Comedy");
+const comedyContainer = document.getElementById("comedy");
+const comedyCard = createCardForEachShow(findComedy);
+comedyContainer.appendChild(comedyCard);
+
+//<-------------------------------------sci-fi------------------------------->
+let findSciFi = findGenres("Science-Fiction");
+const sciFiContainer = document.getElementById("sci-fi");
+const sciCard = createCardForEachShow(findSciFi);
+sciFiContainer.appendChild(sciCard);
+
+//<-------------------------------------romance--------------------------->
+let findRomance = findGenres("Romance");
+const romanceContainer = document.getElementById("romance");
+const romCard = createCardForEachShow(findRomance);
+romanceContainer.appendChild(romCard);
+
+//<-------------------------------------Crime--------------------------->
+let findCrime = findGenres("Crime");
+const crimeContainer = document.getElementById("crime");
+const crimeCard = createCardForEachShow(findCrime);
+crimeContainer.appendChild(crimeCard);
+
+//<-------------------------------------thriller--------------------------->
+let findThriller = findGenres("Thriller");
+const thrillerContainer = document.getElementById("thriller");
+const thrillCard = createCardForEachShow(findThriller);
+thrillerContainer.appendChild(thrillCard);
+
+//------------------------------reality show ------------------------>
+let findReality = findTypeOfShow("Reality");
+const realityContainer = document.getElementById("reality");
+const realityCard = createCardForEachShow(findReality);
+realityContainer.appendChild(realityCard);
+
+//------------------------------international show ------------------------>
+
+let findInternational = newShows.filter((show) => {
+  if (show.language !== "English") {
+    return show;
+  }
+});
+const internationalContainer = document.getElementById("international");
+const intCard = createCardForEachShow(findInternational);
+internationalContainer.appendChild(intCard);
+//<----------------------------------------end of code for slider-------------------->
+
 window.onload = setup;
+
+//-------------------------------//
```

Comment:
It was very attractive and appealing, good comment styles, overall, nothing really to mention to approve Mya excelling job, tiny bit from my side will be variable names but I get overwhelmed with names too and sink in it, 10/10 **********